### PR TITLE
.travis.yml: Use bears master

### PR DESCRIPTION
**Pull request by jayvdb**
Related to https://github.com/coala/meta/issues/123

Also removes the GitCommitBear hack around
https://github.com/coala/coala-bears/issues/1037

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -109,12 +110,8 @@ script:
...
   - python setup.py bdist_wheel
   - pip install ./dist/coala-*.whl
-  - pip install coala-bears[alldeps] --pre -U
-  # https://github.com/coala/coala-bears/issues/1037
-  - >
-    if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then
-      sed -i.bak '/bears = GitCommitBear/d' .coafile
-    fi
+  - curl -fsSL -o coala-bears.zip $BEARS_ZIP_URL/master
+  - pip install coala-bears.zip[alldeps] --pre -U
```
Comment (refeed) [inline .travis.yml]: Btw, I still quite don't understand about this. Doesn't `pip install coala-bears[alldeps] --pre -U` already install the master branch of `coala-bears` ?

Comment (jayvdb) [reply .travis.yml]: No. It installs the pre-release in pypi, from January 2018.

We still are not ready to do pre-releases yet, with coala-utils and dependency-management still needing more work and need to be released first.

**Comment by yukiisbored** [comment]
ack https://github.com/coala/coala/pull/5616/commits/4e2fc06ade80b444b75986812552e1090eb79f7b

**Comment by yukiisbored** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**